.github/workflows/ethereum-ts.yml

```diff
@@ -1,57 +0,0 @@
-name: Ethereum Typescript
-on:
-  push:
-    paths:
```

Comment:
Can you disable this action instead of deleting it. With a comment.

Reply:
I re-added the action and disabled it